### ISSUE TITLE
Align imported PDF content with pages

### ISSUE THREAD

**Issue by MisterE123** [flxzt/rnote#992]
When importing PDF contents, the contents are imported similar to a pasted image, and the headache of aligning the content with the rnote pages is left up to the user.

**Describe the solution you'd like**  
PDF page content should be automatically matched with rnote pages.

**Describe alternatives you've considered**  
Have an "Annotate PDF" that imports the PDF in linear page mode, with all the PDF pages aligned. Adding pages at the end is possible, or inserting new pages between original pdf pages is possible.

**Additional context**

**Comment by silvasch** [comment]
I'm not sure if I understand you correctly, but I think this can already be achieved by selecting `One per Document Page` from `Page Spacing` in the import dialog.

**Comment by flxzt** [comment]
Yes, in addition to that you can use the snap positions option to align the imported Pdf to the page bounds more easily:

![grafik](https://github.com/flxzt/rnote/assets/19841886/5fa6db93-37eb-40e5-ad12-4142f08db91d)

**Comment by MisterE123** [comment]
I haven't tried the snap positions option... whether it works or not, its usage is not clear at all

**Comment by MisterE123** [comment]
Having it be in the import popup?

**Comment by MisterE123** [comment]
Ah, I just realized what it does, it still makes you drag the content to the right place, but now it will snap to the page... that seems convoluted, when all I want is to import a pdf to edit... why can't the importer just put the pdf content on the correct pages, instead of making me drag it?

**Comment by flxzt** [comment]
there now is the option to adjust the document format to the import Pdf page format in the Pdf import dialog
